app/src/main/res/values/translatable.xml

```diff
@@ -258,6 +258,9 @@ THE SOFTWARE.
 	<string name="coverloader_shadow_title">Load artwork from hidden folder</string>
 	<string name="coverloader_shadow_summary">Try to load artwork from \'/sdcard/Music/.vanilla/ARTIST/ALBUM.jpg\'</string>
 
+	<string name="coverloader_inline_title">Load artwork from file tag</string>
+	<string name="coverloader_inline_summary">Try to load a cover stored in files tag via media library</string>
+
```

Comment:
nit: please change this to 'Try to load embedded artwork from file tags' to be more consistent with the existing descriptions.